### PR TITLE
Update hub.json

### PR DESCRIPTION
**Pull request by DipanjanAtRudderLabs**
Updated hub.json with name of new package

### CODE REVIEW

**Comment by dbeatty10** [comment]
This PR itself looks good @DipanjanAtRudderLabs 👍 

But we're going to hold off on merging and deploying this until the project itself is ready -- I sent you over some changes to make via Slack.

**Review by dbeatty10 — APPROVED** [review]
Hi @DipanjanAtRudderLabs this PR looks good and I'll merge it shortly.

One other thing to note that I didn't mention earlier:
- We recommend that package maintainers use as wide of version ranges as possible for any packages they depend upon

Otherwise, a user might not be able to install two different packages at the same time due to version incompatibilities upon a common package (like `dbt_utils`).

e.g., a wider range like this:
```yaml
packages:
  - package: dbt-labs/snowplow
    version: [">=0.7.0", "<0.8.0"]
```

rather than only a single version like this:
```yaml
packages:
  - package: dbt-labs/snowplow
    version: 0.7.3
```

Our recommendation would be for you to try to widen your version range during a subsequent release.

**Comment by DipanjanAtRudderLabs** [comment]
Thanks for the guidance Doug – we will definitely do so in the next release. Thanks & Regards,Dipanjan From: Doug BeattySent: 28 July 2022 03:21To: dbt-labs/hubcapCc: DipanjanAtRudderLabs; MentionSubject: Re: [dbt-labs/hubcap] Update hub.json (PR ***@***.*** approved this pull request.Hi @DipanjanAtRudderLabs this PR looks good and I'll merge it shortly.One other thing to note that I didn't mention earlier:We recommend that package maintainers use as wide of version ranges as possible for any packages they depend uponOtherwise, a user might not be able to install two different packages at the same time due to version incompatibilities upon a common package (like dbt_utils).e.g., a wider range like this:packages:  - package: dbt-labs/snowplow    version: [">=0.7.0", "<0.8.0"]rather than only a single version like this:packages:  - package: dbt-labs/snowplow    version: 0.7.3Our recommendation would be for you to try to widen your version range during a subsequent release.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***> 

	
        
		Virus-free. www.avast.com

**Pull request merged**
